components/tenant-mgt/org.wso2.carbon.tenant.mgt.ui/src/main/resources/web/tenant-mgt/js/tenant_config.js

```diff
@@ -151,7 +151,7 @@ function validateDomain(fld, isPublicCloudSetup) {
     var indexOfDot = domain.indexOf(".");
     var extension = domain.substring(lastIndexOfDot, domain.length);
 
-    var illegalChars = /([^a-zA-Z0-9\._\-])/; // allow only letters and numbers . - _and period
+    var illegalChars = /([^a-z0-9\._\-])/; // allow only letters and numbers . - _and period

```

Comment:
comment should be changed accordingly from "only letters" to "only lower case letters"